### PR TITLE
Add DATA_ROOT

### PR DESCRIPTION
**Pull request by jilljenn**

### CODE REVIEW

File: mangaki/mangaki/settings.py [hunk 1]
```diff
@@ -253,6 +251,10 @@ def get_sentry_handler(config_instance) -> str:
 
 STATIC_ROOT = config.get('deployment', 'STATIC_ROOT', fallback=os.path.join(BASE_DIR, 'static'))
 MEDIA_ROOT = config.get('deployment', 'MEDIA_ROOT', fallback=os.path.join(BASE_DIR, 'media'))
+DATA_ROOT = config.get('deployment', 'DATA_ROOT', fallback=os.path.join(BASE_DIR, 'data'))
+
+DATA_DIR = DATA_ROOT
```
Comment (RaitoBezarius) [inline mangaki/mangaki/settings.py]: FIXME: replace every occurrences of `DATA_DIR` by `DATA_ROOT`

File: provisioning/group_vars/mangaki_standalone [hunk 2]
```diff
@@ -4,6 +4,7 @@ mangaki_source_db_user: '{{ mangaki_db_user }}'
 mangaki_source_db_name: '{{ mangaki_db_name }}'
 mangaki_source_db_password: '{{ mangaki_db_password }}'
 mangaki_source_static_root: '{{ mangaki_web_www_root }}/static'
+mangaki_source_data_root: '{{ mangaki_web_www_root }}/data'
```
Comment (RaitoBezarius) [inline provisioning/group_vars/mangaki_standalone]: We don't want to expose our side-data to the Internet. Rather `/home/{{user}}/data` or `/opt/mangaki/`, something around these lines, sounds good.

File: provisioning/roles/mangaki_source/README.md [hunk 3]
```diff
@@ -85,6 +85,9 @@ mangaki_source_auth_providers: ['twitter', 'facebook']
 # The path to the web root to store static files in (required).
 mangaki_source_static_root: '/var/www/mangaki/static'
 
+# The path to the web root to store data and snapshots in (required).
+mangaki_source_data_root: '/var/www/mangaki/data'
```
Comment (RaitoBezarius) [inline provisioning/roles/mangaki_source/README.md]: I'd prefer `/var/mangaki/data`.

File: mangaki/mangaki/settings.py (hunk 1 above)
Comment (jilljenn) [reply mangaki/mangaki/settings.py]: Plz noes je vais forcément en oublier

Comment (RaitoBezarius) [reply mangaki/mangaki/settings.py]: Juste ajoute un FIXME j'ai dit

File: provisioning/group_vars/mangaki_standalone (hunk 2 above)
Comment (jilljenn) [reply provisioning/group_vars/mangaki_standalone]: Ou directement /data ? Pas trop sale ?

Comment (RaitoBezarius) [reply provisioning/group_vars/mangaki_standalone]: Trop sale

File: provisioning/roles/mangaki_source/README.md (hunk 3 above)
Comment (jilljenn) [reply provisioning/roles/mangaki_source/README.md]: "to store X in" => "where X is stored" non ?

File: mangaki/mangaki/settings.py (hunk 1 above)
Comment (jilljenn) [reply mangaki/mangaki/settings.py]: Done.

File: provisioning/group_vars/mangaki_standalone (hunk 2 above)
Comment (jilljenn) [reply provisioning/group_vars/mangaki_standalone]: Done.

File: provisioning/roles/mangaki_source/README.md (hunk 3 above)
Comment (jilljenn) [reply provisioning/roles/mangaki_source/README.md]: Done.

File: provisioning/roles/mangaki_source/README.md [hunk 4]
```diff
@@ -85,6 +85,9 @@ mangaki_source_auth_providers: ['twitter', 'facebook']
 # The path to the web root to store static files in (required).
 mangaki_source_static_root: '/var/www/mangaki/static'
 
+# The path to the web root to store data and snapshots in (required).
+mangaki_source_data_root: '/var/mangaki/data'
```
Comment (RaitoBezarius) [inline provisioning/roles/mangaki_source/README.md]: Use `/opt/mangaki/data` for consistency with the default vars of standalone.

Comment (jilljenn) [reply provisioning/roles/mangaki_source/README.md]: I really don't get it. The other path is `/opt/mangaki`. I put `/opt/mangaki/data` everywhere.

**Pull request merged**